tools/train.py

```diff
@@ -101,7 +103,11 @@ def main():
                                '"auto_scale_lr.base_batch_size" in your'
                                ' configuration file.')
 
+    # resume is determined in this priority: resume from > auto_resume
```

Comment:
Is that necessary to add `load-from` too?

Reply:
I don't think it's necessary. Since `load-from` may generally use to load pretrain weights from configs, right?

Reply:
@BIGWangYuDong Hi! Still need the `load-from` option?

Reply:
No need to add load-from in this PR.